rts/lra/lra-ai-dashboard/src/main/resources/META-INF/resources/index.html

```
@@ -0,0 +1,103 @@
+  <div id="chat">
+    <div class="msg assistant">Hello! I can help you diagnose and manage your Narayana LRA coordinator.
+
+Try asking:
+• "Show me all active LRAs"
+• "Are there any stuck or failed transactions?"
+• "Why is LRA &lt;paste-lra-id&gt; stuck?"
+• "Cancel LRA &lt;paste-lra-id&gt;"</div>
+  </div>
+  <div id="form">
+    <textarea id="input" rows="2" placeholder="Ask about your LRA coordinator state…"></textarea>
+    <button id="send">Send</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)